apps/SageAccountWeb/package.json

```diff
@@ -23,12 +25,10 @@
     "react-easy-crop": "^5.0.7",
     "react-router-dom": "^5.3.4",
     "sass": "^1.71.1",
-    "synapse-client": "workspace:*",
     "synapse-react-client": "workspace:*",
     "universal-cookie": "^4.0.4"
   },
   "devDependencies": {
-    "@sage-bionetworks/synapse-types": "workspace:*",
```

Comment:
synapse-types has some runtime code, so it may not truly be a 'devDependency'. I moved it to `dependencies`, but this is just for clarity and won't affect anything